webui-page/webui.js

```diff
@@ -19,6 +19,7 @@ function send(command, param){
   request.open("post", path);
 
   request.send(null);
+  setTimeout(status, 500);
```

Comment:
I prefer leaving the `setTimeout()` below at `1000`.

Reply:
sorry, didn't mean to include these local hacks in this PR